source/stylesheets/components/_footer.scss

```diff
@@ -1,10 +1,6 @@
-$desktop-small: new-breakpoint(max-width 1024px);
-$mobile: new-breakpoint(max-width 480px);
-$iPhone5-min-breakpoint: 320px;
-$iPhone5-max-breakpoint: 568px;
-
 .site-footer {
   background-color: $thoughtbot-gray;
```

Comment:
Yeah, it’s a tad strong. I’ll add a commit here for that, since we are buttoning up the whole design a bit.